configurator/src/main/java/org/picmg/configurator/StateSetTabController.java

```diff
@@ -121,4 +165,13 @@ void onSaveAsChangesAction(ActionEvent event) {
 	public void initialize(URL location, ResourceBundle resources) {
 		//TODO: fill out with needed initialize
 	}
+
+	public boolean isValid() {
+		if(stateSetVendorNameTextField.isVisible()) return  false;
+		if(stateSetVendorIANA.isVisible()) return  false;
+		if(stateSetId.isVisible()) return  false;
+		if(this.getWorkingData().getOemStateValueRecords() != null && this.getWorkingData().getOemStateValueRecords().size() == 0) return false;
+		return true;
+	}
+
 }
```

Comment:
Since this is part of validation, could you add disabling the save buttons when any input is invalid? I created a method in the sensors and effecters tab controllers called setSaveAvailability which you may copy (if so, be sure to match how it is called in those other controllers)

Reply:
Yes.  isValid checks for the  all validations based on that I am enabling/disabling SaveChangesButton in the corresponding action for the field like the following saveChangesButton.setDisable(!isValid());